dockerfiles/base/scripts/base/library.sh

```diff
@@ -256,21 +256,28 @@ docker_compose() {
   fi
 }
 
+# Return srandom character alphanumeric string (upper and lower case) of length $1
+generate_random_string() {
```

Comment:
yes I would say that we only let code really used in master. Dead code is usually deleted.
you may add the code to the pull request so if you need to find it again, you could get it.
Or paste it to a gist